src/main/java/bennett/citibike/json/CitiBikeService.java

```diff
@@ -0,0 +1,17 @@
+package bennett.citibike.json;
+
+import bennett.citibike.json.StationResponse;
+import bennett.citibike.json.StatusResponse;
+import io.reactivex.rxjava3.core.Single;
+import retrofit2.http.GET;
+
+public interface CitiBikeService {
+
+    @GET("en/station_information.json")
```

Comment:
```suggestion
    @GET("station_information.json")
```

add the "en/" to the baseUrl